tests/cases/user_test.py

```diff
@@ -568,7 +568,7 @@ def testEmailVerification(self):
 
         # cannot login without verifying email
         resp = self.request('/user/authentication', basicAuth='user:password')
-        self.assertStatus(resp, 403)
+        self.assertStatus(resp, 401)
         self.assertTrue(resp.json['extra'] == 'emailVerification')
 
```

Comment:
It just makes our `AccessException`s behave uniformly in all cases, whereas before, authentication failure was being treated as a special case.